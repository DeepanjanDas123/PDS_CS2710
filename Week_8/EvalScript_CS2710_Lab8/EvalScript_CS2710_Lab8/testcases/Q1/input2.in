34
31
push_back 14
push_front 30
isEmpty
pop_front
push_back 20
push_back 36
push_back 24
push_front 36
pop_front
pop_front
push_back 3
push_front 51
isEmpty
pop_front
pop_front
push_back 99
pop_front
isEmpty
push_front 62
push_front 41
pop_back
isEmpty
pop_back
pop_back
pop_back
push_front 19
isEmpty
pop_back
isEmpty
pop_back
pop_back