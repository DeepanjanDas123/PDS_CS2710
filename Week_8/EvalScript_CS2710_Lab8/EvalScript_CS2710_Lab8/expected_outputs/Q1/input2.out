False
30
36
14
False
51
20
36
False
99
False
3
24
62
False
41
False
19
Deque is empty